### ISSUE TITLE
Child components don't update

### ISSUE THREAD

**Issue by DrogoNevets** [djorg83/react-bootstrap-sweetalert#30]
I am trying to use react-treebeard to put a tree item inside a sweetalert.

whilst the initial render works fine, it doesn't redraw when clicking  the nodes.

it does however update tree views linked to the same data, so the functions etc are being called, just doesnt appear to redraw?

```
<SweetAlert title="Good job!" onConfirm={this.onConfirm.bind(this)}>
    <Treebeard
        data={this.state.treeData}
        onToggle={this.onToggle.bind(this)}
    />
</SweetAlert>
```

**Comment by djorg83** [comment]
Yeah, this is a known issue.  I have dealt with this in the past by hiding/showing the alert when something inside the content needs to be updated.  I'll see if I can fid a better solution.

**Comment by djorg83** [comment]
This issue is resolved via usage of `props.dependencies`, available now in `v5.2`.
